lib/ex_machina.ex

```diff
@@ -49,6 +49,33 @@ defmodule ExMachina do
       def build_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.build_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def create(_) do
+        raise_error_helper("create/1", "insert/1")
+      end
+
+      def create(_, _) do
+        raise_error_helper("create/2", "insert/2")
+      end
+
+      def create_pair(_, _) do
+        raise_error_helper("create_pair/2", "insert_pair/2")
+      end
+
+      def create_list(_, _, _) do
+        raise_error_helper("create_list/3", "insert_list/3")
+      end
+
+      defp raise_error_helper(old_function, new_function) do
```

Comment:
In favor as well, but how about `raise_function_replaced_error`. Given the `old_function` & `new_function` params?